conformance/service_import.go

```diff
@@ -87,6 +97,44 @@ var _ = Describe("", func() {
 				reportNonConformant(fmt.Sprintf("The value %q is not a valid IP", serviceImport.Spec.IPs[0])))
 		})
 
+	Specify("A ServiceImport should only exist as long as there's at least one exporting cluster",
+		Label(RequiredLabel), func() {
+			requireTwoClusters()
+
+			AddReportEntry(SpecRefReportEntry, "https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/1645-multi-cluster-services-api/README.md#importing-services")
+
+			serviceImport := t.awaitServiceImport(&clients[0], t.helloService.Name, func(serviceImport *v1alpha1.ServiceImport) bool {
+				return true
+			})
+			Expect(serviceImport).NotTo(BeNil(), "ServiceImport was not found")
+
+			By(fmt.Sprintf("Exporting the service on the second cluster %q", clients[1].name))
+
+			t.deployHelloService(&clients[1], newHelloService())
+			t.createServiceExport(&clients[1])
+
+			// Sanity check and to also wait a bit for the second cluster to export. There's no deterministic way to tell if/when the
+			// second cluster has finished exporting other than utilizing different service ports in each cluster but that's already
+			// done in another test case.
```

Comment:
`other than utilizing different service ports in each cluster but that's already done in another test case.` 
Is that a future improvement that could be done here too? AFAICT this is not checked in the code below, if that's the case maybe this could be marked as FIXME/TODO or something like that?

Reply:
I wanted to avoid this test relying on the service port behavior as that is tested on L168 below, otherwise if there's an issue with service port compliance than this test would also fail orthogonally.

  > Is that a future improvement that could be done here too? AFAICT this is not checked in the code below, if that's the case maybe this could be marked as FIXME/TODO or something like that?

I assume you're referring to the L168 test that doesn't unexport and verify the ports are adjusted correctly. I could add that but I also was wary of that failing orthogonally if the SI was unexpectedly deleted.

Reply:
Ah ok understood! Maybe it would be nice to clarify this in the comment as the part `but that's already done in another test case.` is not 100% clear to me as to why this was not done without your additional explanation?


Maybe something like: `second cluster has finished exporting other than utilizing different service ports in each cluster but the scope of this test is not about testing a valid port merging implementation.`? (Feel free to update this, it's just a proposal)